data/ecosystem/adopters.yaml

```
@@ -81,6 +110,24 @@
   components: [Java, Python]
   reference: ''
   contact: gael.jourdan-weil@kelkoogroup.com
+- name: Lockheed Martin
+  url: https://www.lockheedmartin.com/
+  components: [Collector]
+  reference: 'https://www.cncf.io/case-studies/lockheed-martins-enterprise-operations-chose-otel-for-better-observability/'
+  referenceTitle: CNCF case study
+  contact: ''
+- name: loveholidays
+  url: https://www.loveholidays.com/
+  components: [Collector]
+  reference: 'https://www.cncf.io/case-studies/reducing-mttd-and-increasing-observability-with-linkerd-at-loveholidays/'
+  referenceTitle: CNCF case study
+  contact: ''
+- name: Meltwater
+  url: https://www.meltwater.com/
+  components: [Collector]
+  reference: 'https://www.cncf.io/case-studies/meltwaters-live-migration-to-cilium-for-richer-features/'
+  referenceTitle: CNCF case study
+  contact: ''
 - name: MercadoLibre
```

Mind if I use this PR to sneak in the missing space? 🙃 

```suggestion
- name: Mercado Libre
```